examples/graceful-shutdown/graceful-shutdown/server.go

```diff
@@ -1,4 +1,4 @@
-// +build go1.8
+// +build go1.11
```

Comment:
why change to `go1.11`?

Reply:
I am using  v1.11. If it is not suitable， I will change it back to v1.8.

Reply:
yes, you should not update.